data_steward/resources.py

```diff
@@ -125,3 +139,15 @@ def cdm_schemas(include_achilles=False, include_vocabulary=False):
             if include_table:
                 result[table_name] = schema
     return result
+
+
+def hash_dir(in_dir):
+    """
+    Generate an MD5 digest from the contents of a directory
+    """
+    fs = os.listdir(in_dir)
+    hash_obj = hashlib.md5()
```

Comment:
According to codacy, this is an insecure hash function.  Is this just to generate a version string?

Reply:
I changed it to sha1